tests/plugin-test/css-extract/cases/dependOn-multiple-files-per-entry/expected/entry1.css

```diff
@@ -1,8 +1,8 @@
-.styleB {
-  background: blue;
-}
-
 .styleA {
   background: red;
 }
 
+.styleB {
+  background: blue;
+}
+
```

Comment:
Webpack/Rspack legacy code-splitting bug, you can see entry modules from config is A then B